src/frontend.jl

```diff
@@ -54,17 +54,22 @@ macro progress(ex)
 end
 
 """
-    progress(x = [0..1])
+    progress(x = [0..1], [message])
 
-Set Atom's progress bar to the given value.
+Set Atom's progress bar to the given value. Displays `message` as a tooltip.
 """
 progress(x::Void = nothing) = Atom.msg("progress", "indeterminate")
 
 progress(x::Real) =
-  Atom.msg("progress", (x < 0.01 ? nothing :
+  Atom.msg("progress", (x < 0.01 ? "indeterminate" :
                         x > 1 ? 1 :
                         x))
 
+progress(x::Real, message::String) =
+  Atom.msg("progress", (x < 0.01 ? "indeterminate" :
```

Comment:
No, the message will be displayed no matter what. 
Indeterminate just means the progress bar will display the moving gray stripes you already see e.g. on julia startup instead of the blue colored progress bar. The `x < 0.1` is more or less a hack to show that something is going on even though you wouldn't be able to see the progress because the progress bar is to small.